string name
int64[] array
---
string status